### PR TITLE
Added new logo and icon

### PR DESCRIPTION
**Pull request by sevarsh**
New banner and icon were designed and added to the project. 
Template's logo and icon were removed from the project, the names of template's logo and icon files were replaced in the code of index.html, index.js and app.js.

### CODE REVIEW

**Review by Tumanov-Viatcheslav — APPROVED** [review]
Meaning of this change 'Limitations'->'Limitationsя' is not clear

**Review by Tumanov-Viatcheslav — APPROVED** [review]
Clear

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# MapViewer
+  # MapViewer
 ![](http://www.visit-petersburg.ru//media/uploads/%D0%BC%D0%B0%D1%80%D1%88%D1%80%D1%83%D1%82%D1%8B.jpeg)
 ## Puprose
 Java Script project to work with maps
-## Limitations
+## Limitationsя
```
Comment (Tumanov-Viatcheslav) [inline README.md:5]: Meaning of change?

**Review by Tumanov-Viatcheslav — APPROVED** [review]

**Comment by sevarsh** [comment]
The new symbol in "README.md" is a misprint, i'm sorry. I'd corrected this line is the next commit in the other branch (MapViewer-13). I'm not sure, if it's right, but i don't see the sense in commiting to delete just one letter.

**Pull request merged**